homeassistant/components/auth/__init__.py

```diff
@@ -604,6 +604,7 @@ async def websocket_delete_refresh_token(
 @websocket_api.websocket_command(
     {
         vol.Required("type"): "auth/delete_all_refresh_tokens",
+        vol.Optional("token_type"): cv.string,
```

Comment:
```suggestion
        vol.Optional("token_type"): cv.string,
        vol.Optional("remove_current_token", default=True): bool
```

Reply:
If the most common thing to do is to log everything out but yourself, should the default be False?

Reply:
`True` was to avoid a 'technically' breaking change, but if we aren't worried about other consumers of the API, defaulting to `False` would be better for this case.

Reply:
I guess we don't need to break it if we keep the default as True.

Reply:
I have added this option (w/ default true), and will leverage it from the frontend.